docker/infra/adguardhome.yaml

```
@@ -1,6 +1,12 @@
 # AdGuard Home is a network-wide software for blocking ads and tracking. After you set it up, it'll cover all your home devices, and you won't need any client-side software for that
 #
-# To fix "listen tcp4 0.0.0.0:53: bind: address already in use" error on Ubuntu: disable `DNSStubListener` (see Ansible)
+# To fix "listen tcp4 0.0.0.0:53: bind: address already in use" error on Ubuntu: disable `DNSStubListener` (see Ansible).  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)